smoke-test-samples/common/src/main/java/io/javaoperatorsdk/operator/sample/CustomServiceReconciler.java

```diff
@@ -15,7 +15,7 @@
 
 /** A very simple sample controller that creates a service with a label. */
 @ControllerConfiguration
-public class CustomServiceReconciler implements Reconciler<CustomService> {
+public class CustomServiceReconciler implements Reconciler<CustomService>, Cleaner<CustomService> {
```

Comment:
If it's a very simple sample like the description above says, shouldn't it use k8s garbage collector and not implement Cleaner ? I mean it should be easier to do it that way than implementing Cleaner.

Reply:
will remove cleaner, thx!